pkg/handler/eventhandler.go

```diff
@@ -87,6 +87,12 @@ var _ EventHandler = Funcs{}
 // Funcs implements eventhandler.
 type Funcs = TypedFuncs[client.Object, reconcile.Request]
 
+// NewFuncsWithLowPriority creates a Funcs Handler wrapped in
+// WithLowPriorityWhenUnchanged
+func NewFuncsWithLowPriority(u Funcs) EventHandler {
```

Comment:
The naming of NewFuncsWithLowPriority vs WithLowPriorityWhenUnchanged seems inconsistent (New vs With)